backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepository.java

```diff
@@ -52,6 +52,7 @@
 	private Map<UUID, List<Token>> shopIdToListOfTokensMap = new HashMap<>();
 	private Map<UUID, AtomicInteger> shopIdToLastAllotedTokenMap = new HashMap<>();
 	private Map<String, ShopOwner> shopOwnerMap = new HashMap<>();
+	private final static long WAITING_TIME_MINS = 4L;
```

Comment:
Should we not mention it here then that it is per customer??

Reply:
This name is what Priyanka had asked me to use maybe because if we have mentioned it at one place (along with a comment) already, the idea or logic residing in the developer's mind becomes clear to a new reader. This is what I can understand.